setzer/dialogs/bibtex_wizard/pages/page_create_new_entry.py

```diff
@@ -59,8 +60,8 @@ def __init__(self, document_types):
         PageView.__init__(self)
         self.document_types = document_types
             
-        self.header.set_text('Choose a document type')
-        self.headerbar_subtitle = 'Step 1: Choose document type'
+        self.header.set_text(_('Choose a document type'))
+        self.headerbar_subtitle = _('Step') + '1: ' + _('Choose a document type')
```

Comment:
'Step' appears quite often, so it's better to separate it from the number and the other string, this way the translators only have to translate it once.